cross_win32.txt

```diff
@@ -6,7 +6,7 @@ c = 'i686-w64-mingw32-gcc'
 cpp = 'i686-w64-mingw32-g++'
 ar = 'i686-w64-mingw32-ar'
 strip = 'i686-w64-mingw32-strip'
-pkg-config = 'i686-w64-mingw32-pkg-config'
+pkgconfig = 'i686-w64-mingw32-pkg-config'
```

Comment:
Seems unrelated to this PR

Reply:
This was actually required for 64-bit Ubuntu-based machines to cross-build Win32 32-bit binaries; the build system was checking for 32-bit versions of packages only without that revert.

Reply:
Fixed. Used ```meson``` from ```pip``` instead of ```apt``` as the ```pip``` version is FAR more up-to-date than what's currently packaged as a .deb file.